blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
```

Comment:
I think the word "stringify" is confusing. Namely many Snap<em>!</em> users aren't familiar with the term "string", but also, in my mind at least "stringify" implies are more generic use than Scratch Blocks syntax.